ci/ci/ci.py

```diff
@@ -113,7 +113,7 @@
         if hasattr(pr.batch, 'id'):
             status = await pr.batch.status()
             for j in status['jobs']:
-                j['duration'] = humanize.naturaldelta(Job.total_duration(j))
+                j['duration'] = humanize.naturaldelta(datetime.timedelta(milliseconds=Job.total_duration_mscs(j)))
```

Comment:
There's a typo in `mscs`

Reply:
Fixed.  I added a timedelta humanize utility function a few other fixes for the tests.